wgsl/index.bs

```diff
@@ -41,11 +41,32 @@ thead {
 
 ## Goals ## {#goals}
 
- * Trivially convertable to SPIR-V
- * Constructs are defined as normative references to their SPIR-V counterparts
- * All features in [SHORTNAME] are directly translatable to SPIR-V. (No polymorphism, no general pointers, no overloads, etc)
- * Features and semantics are exactly the ones of SPIR-V
- * Each item in this spec *must* provide the mapping to SPIR-V for the construct
+In no particular order:
+
+ * Text-based
```

Comment:
Text-based seeems redundant when there's also "readable and writable by programmers".  Can we imagine a WGSL spec that's readable and writable by programmers, but isn't also text based?